packages/vue/src/components/molecules/SfBar/SfBar.vue

```diff
@@ -1,38 +1,38 @@
 <template>
   <div class="sf-bar">
     <slot name="back">
-      <div class="sf-bar__icon">
-        <SfIcon
-          v-if="back"
-          icon="chevron_left"
-          size="14px"
-          role="button"
-          aria-label="back"
-          @click="$emit('click:back')"
-        />
-      </div>
+        <button v-focus class="sf-bar__icon">
```

Comment:
maybe we can use `SfButton` here, wdyt? @justyna-13

Reply:
If @pspaczek agrees on the new modifier to button then it'll be better with SfButton here but if not, then I'd rather stay with button tag - will be less changes to do :)